### ISSUE TITLE
A11y issues with queue - the second modal is not announced

### ISSUE THREAD

**Issue by limonte** [limonte/sweetalert2#563]
Separated from #544 

> A11y issues with queue - the second modal is not announced, and for modals with form controls, those controls lack an explicit accessible name/label.

**Comment by limonte** [comment]
Duplicate of #564
